src/template/README.md

```diff
@@ -21,7 +21,7 @@ Run the app in development mode:
 
 `npm start`
 
-Open [http://localhost:5000](http://localhost:5000) to view it in the browser.
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
on the .env sample in the readme there is also a reference to port 5000

Reply:
thanks I totally missed that one!